data_science/specs/cohort_spec.rb

```diff
@@ -0,0 +1,100 @@
+require 'rspec'
+require 'json'
+require_relative '../cohort.rb'
+
+RSpec.describe 'Cohort' do
+  let (:test_data) { JSON.parse(File.read('../test_data.json')) }
+  let (:cohort_a) { test_data.select { |v| v['cohort'] == 'A' } }
+
+  before :each do
+    @cohort = Cohort.new(name: 'A', data: cohort_a)
```

Comment:
Standard idiom in RSpec is to use `subject`:

```
subject { Cohort.new(name: 'A', data: cohort_a) }
```

Try not to use `before` to initialize instances since `let` does a similar thing in a better way. `before` is still useful for mutation based setup.